gemfirexd/client/src/main/java/com/pivotal/gemfirexd/internal/client/net/NetConnection.java

```diff
@@ -1341,7 +1341,7 @@ void rdbAccessed(int svrcod,
     public java.sql.CallableStatement getBucketToServerDetails_;
 
     public static final String BUCKET_AND_SERVER_PROC_QUERY =
-      "call SYS.GET_BUCKET_TO_SERVER_MAPPING(?, ?)";
+      "call SYS.GET_BUCKET_TO_SERVER_MAPPING2(?, ?)";
```

Comment:
Yes for client backward compatibility (which is what I was asking the other day if we want to have GFXD client compatibility or not). Can remove if Jags confirms that it is not important.